src/Sylius/Bundle/RbacBundle/Security/RbacVoterInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\RbacBundle\Security;
+
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+/**
+ * @author Christian Daguerre <christian@daguer.re>
+ */
+interface RbacVoterInterface extends VoterInterface
```

Comment:
I'm not sure about this empty interface, do we really benefit from it? We ans Symfony use `VoterInterface` method only.